ansible/roles/neutron/templates/eswitchd.conf

```diff
@@ -0,0 +1,2 @@
+[DAEMON]
+fabrics = ibphysnet:{{ neutron_mlnx_interface }}
```

Comment:
Should this take a dictionary with mappings from physnet to interface?

Reply:
Yes makes more sense to do this as a mapping.

Reply:
Although i was trying to use the same pattern as physical_interface_mapping in `ansible/roles/neutron/templates/ml2_conf.ini.j2`